### ISSUE TITLE
[6.2 RC1] Error: Undefined variable: "$primary-color"

### ISSUE THREAD

**Issue by JeremyEnglert** [zurb/foundation-sites#8240]
When attempting to compile the settings.scss file with 6.2, I get the following error (using Gulp): 
```
Error: Undefined variable: "$primary-color".
        on line 140 of assets/scss/_settings.scss
>> $anchor-color: $primary-color;
```
Since $primary-color has been replaced with the color palette, how does the Sass make the connection between $primary-color and the new variable? (Sorry for the newb question) 

If I create a variable for $primary-color, it resolves the issue, but then a new error comes up:
```
Error: Undefined variable: "$progress-height".
        on line 388 of assets/scss/_settings.scss
>> $meter-height: $progress-height;
```
The second error looks like it is caused by $progress-height being declared after $meter-height tries to use it.

**Comment by gakimball** [comment]
The slider variable error is fixed via 70fc73beabab0819ac87d0cc46e7ae11dfe6d049

As for the color issue, we added a shim mixin that translates the `$foundation-palette` values into the old color variables for you. It's called `add-foundation-colors()`. Instructions on adding it will be in the 6.2 migration guide.

You can put it at the bottom of the global section of your settings file:

```scss
// 1. Global
// ---------

$foundation-palette: ();

@include add-foundation-colors;
```

**Comment by JeremyEnglert** [comment]
Whoops - sorry. I was using an older version of 6.2. Working great now.
